src/WebJobs.Script/OutOfProc/Rpc/Configuration/WorkerConfigFactory.cs

```diff
@@ -113,35 +121,38 @@ internal void AddProvidersFromAppSettings()
 
         internal void AddProvider(string workerDir)
         {
-            try
+            using (_metricsLogger.LatencyEvent(MetricEventNames.AddProvider))
```

Comment:
this would more useful if the metric name includes workerDir

Reply:
Agreed, added this.